playbooks/adhoc/uninstall_openshift.yml

```diff
@@ -268,6 +268,7 @@
     with_subelements:
     - "{{ directories.results | default([]) }}"
     - files
+    failed_when: false
```

Comment:
What was this fixing?

Reply:
If this path is mounted, the rm will fail. This ignores this failure while still rm the rest of the content. ie... now best effort.

Reply:
@sdodson Added comment reflecting the need.

Reply:
ahh yeah, of course, thanks